api/proto/config/storage.proto

```diff
@@ -25,21 +25,23 @@ enum DsType {
 }
 
 message DatastoreConfig {
-	string id = 100;
+  string id = 100;
 
-	DsType dType = 1;
+  UUIDandVersion uuidandversion = 101;
```

Comment:
Ditto

Reply:
"id" will be deprecated once this is deployed everywhere.
Till then we can maintain backward compatibility.

Reply:
That would require 1) comment in the proto file e.g., // XXX to be deprecated for the id and 2) code in EVE to use the UUIDandVersion. Latter would be best done once Srinibas code has merged.

But is there code in zedcloud to increment the version any time there is a change?

Reply:
Same reply for first 2 points as above.

Yes, zedcloud bumps up version of an object if there is any update to that object.